lib/libcasper/services/cap_fileargs/cap_fileargs.3

```diff
@@ -60,52 +60,55 @@
 .Ft "char *"
 .Fn fileargs_realpath "fileargs_t *fa" "const char *pathname" "char *reserved_path"
 .Sh DESCRIPTION
-The library is used to simplify Capsicumizing a tools that are using file system.
-Idea behind the library is that we are passing a remaining
+The library is used to simplify Capsicumizing tools that are using file system.
+The idea behind the library is that we are passing the remaining of
```

Comment:
```suggestion
The
.Nm
library is used to simplify Capsicumizing tools
which use the file system by passing the remaining of
```
Two things to point out:
+ I find "the remaining of argc and argv" to be unclear?
+ Please insert the line `.Nm cap_fileargs` after line 28.

Reply:
Rephrased "the remaining of `argc` and `argv`" to make it concise.